### ISSUE TITLE
Issues with macOS keyboard bindings

### ISSUE THREAD

**Issue by karniv00l** [TNorbury/command_palette#22]
At the moment, keyboard bindings don't work very good on mac, makes this package almost unusable. 

Solution will be to use `SingleActivator` as per comment: https://github.com/flutter/flutter/issues/97589#issuecomment-1058750417

**Comment by TNorbury** [comment]
Interesting, I seemed to have missed this before. To confirm, the issue you're describing is when you open the palette and then press and an arrow key, and then the arrow key doesn't work (but will work if you try again after a few seconds)?

**Comment by karniv00l** [comment]
Using example in this repo, I think the simplest way to reproduce the issue:

1. Open console (CMD+K)
2. Close console
3. Try different combination like CMD+P
4. Now CMD+K is not working anymore

Also, none of the action shortcuts in the example work

**Comment by TNorbury** [comment]
Gotcha. I got a solution working for the above, need to do some testing, but it looks like everything is working as expected on Mac web.

As for the action shortcuts, those are purely visual. My intention is that the app developer would implement their app specific shortcuts, and then just display the relevant shortcuts. Reason being is I don't intend for this package to be a keyboard shortcut manger, nor do I want to take away a developer's control over their shortcuts. But I recognize that often times the actions in a command palette have a shortcut that can also be used to invoke them. As such I added the ability to show which shortcut could be invoked to use that action (without having to use the command palette). That being said I am open to reconsideration.
